src/NewTools-SettingsBrowser/SpNotebookPresenter.extension.st

```diff
@@ -0,0 +1,19 @@
+Extension { #name : 'SpNotebookPresenter' }
+
+{ #category : '*NewTools-SettingsBrowser' }
+SpNotebookPresenter >> hasPageNamed: aTitle [
+
+	^ self pages anySatisfy: [ : page | 	page title = aTitle ]
+]
+
+{ #category : '*NewTools-SettingsBrowser' }
+SpNotebookPresenter >> hasPages [
+
+	^ self pages isNotEmpty
+]
+
+{ #category : '*NewTools-SettingsBrowser' }
+SpNotebookPresenter >> pageNamed: aTitle [
+
+	^ self pages detect: [ : page | page title = aTitle ]
+]
```

Comment:
I'd prefer this to be directly in the Notebook presenter

Reply:
Agree. Done in https://github.com/pharo-spec/Spec/pull/1636